mitmproxy/dump.py

```diff
@@ -248,7 +248,7 @@ def _echo_request_line(self, flow):
         else:
             client = click.style("[replay]", fg="yellow", bold=True)
 
-        method = flow.request.method
+        method = flow.request.data.method
```

Comment:
I tested out the `clean_bin` like so:

```python
strutils.clean_bin(u'\x07')
```
and it doesn't produce the beep (which `print(u'\x07'` does.) This means all is good.

[This commit](https://github.com/mitmproxy/mitmproxy/pull/1299/commits/de96e64732dd26d32aee0de0a1bed35fe4732e51) will handle control pictures, I hope.